### ISSUE TITLE
311 Requests / updates

### ISSUE THREAD

**Issue by sebastianebarb** [CityOfBoston/design-team-workflow#428]
- Pick from multiple addresses based on search https://github.com/CityOfBoston/311/issues/579
- Multiple images per case https://github.com/CityOfBoston/311/issues/505
- 404 page https://github.com/CityOfBoston/311/issues/19

**Comment by sebastianebarb** [comment]
Fin Hopkins
[10:32 AM] 
@sebastianebarb I think I want to revisit the multiple images stuff. The bulk of requests have 1 image, and some have 2 (report and closed). Above that I think doesn’t come up very much?

I’m wondering about:
 - 1 image: what we have now
 - 2 images: images side-by-side, with map as a rectangle underneath (probably 1/2 height of the images)
 - 3 images: 3 square images across, map underneath
 - 4+ images: 2 columns of images, map underneath (or alternately map filing in the last square of the grid if the # of images is odd?)


[10:32] 
Also, right now mobile just shows the map then the images in a column. I think reasonable to keep that behavior?


[10:33] 
Basically  what I’m solving for above ^^^ is the case of 2 images, being able to see them both without clicking the thumbnail.

**Comment by sebastianebarb** [comment]
<img width="322" alt="screen_shot_2017-08-29_at_4 29 39_pm_720" src="https://user-images.githubusercontent.com/20642299/29875876-34851ca4-8d69-11e7-873a-0597a80f7154.png">
